src/test/scala/cromwell/MainSpec.scala

```diff
@@ -170,15 +164,16 @@ class MainSpec extends FlatSpec with Matchers with BeforeAndAfterAll with TimeLi
     }
   }
 
-  it should "run reading options" in {
-    testWdl(ThreeStep, optionsJson = """{ "default_backend": "BAD_BACKEND" }""") { wdlAndInputs =>
-      val wdl = wdlAndInputs.wdl
-      val inputs = wdlAndInputs.inputs
-      val options = wdlAndInputs.options
-      traceErrorWithExceptionRun(wdl, inputs, options)(
-        "WorkflowManagerActor: Workflow failed submission: bad_backend is not a recognized backend") should be(1)
-    }
-  }
+  // FIXME: Currently disabled as there's no way to test this until the new validation stuff goes in
+//  it should "run reading options" in {
+//    testWdl(ThreeStep, optionsJson = """{ "default_backend": "BAD_BACKEND" }""") { wdlAndInputs =>
+//      val wdl = wdlAndInputs.wdl
+//      val inputs = wdlAndInputs.inputs
+//      val options = wdlAndInputs.options
+//      traceErrorWithExceptionRun(wdl, inputs, options)(
+//        "WorkflowManagerActor: Workflow failed submission: bad_backend is not a recognized backend") should be(1)
+//    }
+//  }
```

Comment:
What do you think about putting the code for this test in the description on the JIRA ticket instead?  If you don't want to I don't care too much about it... I get a sinking feeling every time I see a test commented out because I feel that they rarely get uncommented.  Like, if I took the ticket to implement this feature, I may not even know this commented out test exists or applies to me.

Maybe even just a mention in the description of the ticket that this test exists and should be uncommented and used so it doesn't get forgotten?